src/Forcing/relaxation.jl

```diff
@@ -0,0 +1,102 @@
+@inline zerofunction(args...) = 0
+@inline onefunction(args...) = 1
+
+"""
+    struct RelaxingFunction{R, M, T}
+
+Callable object for restoring fields to a `target` at
+some `rate` and within a `mask`ed region in `x, y, z`. 
+"""
+struct RelaxingFunction{R, M, T}
+      rate :: R
+      mask :: M
+    target :: T
+end
+
+"""
+    Relaxation(; rate, mask=onefunction, target=zerofunction)
+
+Returns a `SimpleForcing` that restores a field to `target(x, y, z, t)`
+at the specified `rate`, in the region `mask(x, y, z)`.
+
+Example
+=======
+
+* Restore a field to a linear z-gradient everywhere on a timescale of one minute.
+
+```julia
+julia> restore_stratification = Relaxation(; rate = 1/60, target = (x, y, z, t) -> z)
+```
+"""
+Relaxation(; rate, mask=onefunction, target=zerofunction) =
```

Comment:
Hmmm, why not call this `RelaxingFunction`? It's a constructor for `RelaxingFunction` right? Hmmm, well it constructs a `SimpleForcing` so maybe not. I guess the user only interacts with `Relaxation`.

Reply:
Right! They are different: `RelaxationFunction` is the function itself, `Relaxation` is the forcing. We could also call it `RelaxingForcing`. I just liked the idea of adding some `Relaxation` to my scripts.

Reply:
Indeed! `RelaxationFunction` is the function (actually, a callable object), while `Relaxation` is the forcing. I could also call it `RelaxingForcing` or `SimpleRelaxingForcing`. I just liked the idea of adding some `Relaxation` to my scripts.